<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyDebugDialog</class>
 <widget class="QWidget" name="MyDebugDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>857</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_title">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #FFFFFF;
background-color: #0AA9E3;
min-height: 30px;
font: 18px;</string>
       </property>
       <property name="text">
        <string>Debug Info</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_close">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton
{
border: 0px;
padding: 3px;
background-color: #0AA9E3;
}
QToolButton:hover
{
background-color: #E81123;
}
QToolButton:pressed
{
background-color: #B2241C;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resource/resource.qrc">
         <normaloff>:/layout/layout/close.png</normaloff>:/layout/layout/close.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="widget_background" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="TabBar" name="tabBar" native="true"/>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="page_debug">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QTextEdit" name="textEdit">
            <property name="lineWrapMode">
             <enum>QTextEdit::NoWrap</enum>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_clear">
              <property name="text">
               <string>Clear</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_setting">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QGroupBox" name="groupBox_category">
            <property name="title">
             <string>Logging Category</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QListWidget" name="listWidget_category">
               <property name="alternatingRowColors">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="ComboBox" name="comboBox_category">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="editable">
                  <bool>true</bool>
                 </property>
                 <item>
                  <property name="text">
                   <string>qt_message</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>qt_system</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>qt_motion_state</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>qt_camera_status</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>qt_camera_press</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>qt_camera_release</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>qt_camera_doubleclick</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>qt_playback_split_time</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>qt_playback_realtime</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_add">
                 <property name="text">
                  <string>Add</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_delete">
                 <property name="text">
                  <string>Delete</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>combobox.h</header>
  </customwidget>
  <customwidget>
   <class>TabBar</class>
   <extends>QWidget</extends>
   <header>tabbar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resource/resource.qrc"/>
 </resources>
 <connections/>
</ui>
